standards/SPIFFE_Federation.md

```diff
@@ -307,6 +307,56 @@ As described in the [Trustworthiness of the Bundle Endpoint Server](#74-trustwor
 
 Finally, it should be noted that the "links" in this chain are formed by individual HTTPS request operations (against the various bundle endpoint servers in the chain), and these operations are likely occurring at different times. SPIFFE bundle endpoint clients should be logging these HTTPS request operations, and administrators should take care to preserve these logs for future forensic analysis if necessary.
 
+### 7.7 Processing claims asserted by foreign trust domains
+
+SPIFFE Federation allows workloads in one trust domain to authenticate SVIDs, and all the claims therein, issued by other trust domains.  By an "authentic" claim, we mean specifically that it can be safely assumed that the foreign trust domain authority _asserted_ the claim.  This is not the same thing as saying that the claim is _true_, or that it is _correctly interpreted_.
+
+Operators should use caution and good judgement in evaluating if and how they should process claims asserted in SVIDs from foreign trust domains.
```

Comment:
Should we nudge folks in the direction of allowlisting, operator-driven agreement, etc? Here's a poorly-thought-through example of what that could look like

```suggestion
Operators should use caution and good judgement in evaluating if and how they should process claims asserted in SVIDs from foreign trust domains. At a minimum, the claims to be utilized and their exact meaning should be agreed upon amongst the operators of all involved trust domains beforehand. Unless otherwise agreed upon, it is strongly recommended that claims from foreign trust domains not be processed.
```

Reply:
Added my own attempt at this, PTAL